package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "3.8.10",
+  "version": "3.8.10a",
```

Comment:
Update to 3.8.10 release branch.

I will have to commit these same changes to main branch (which may be easier than merging this release branch into main branch).